plugins/woocommerce/.gitignore

```diff
@@ -30,3 +30,5 @@ i18n/languages/woocommerce.pot
 
 # Environment files
 .wp-env.override.json
+
+.idea
```

Comment:
Please do not include this file in this PR.

Btw, you can add this to your global .gitignore file to stop it coming up in diffs

Reply:
Nice catch, @opr! I pushed the revert change! 🙌